### ISSUE TITLE
[BUG] <CartProvider> SSR warning

### ISSUE THREAD

**Issue by beamercola** [Shopify/hydrogen-ui#65]
**Describe the bug**
Using <CartProvider> in _app throws the error:

> Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.

```
- "@shopify/hydrogen-react": "^2022.10.2"
- "next": "^13.0.3"
- Node v16.17.1
```

**Comment by beamercola** [comment]
it works!

**Comment by frehner** [comment]
Great! It was just released in 2022.10.3 👍

**Comment by beamercola** [comment]
@frehner I swear this went away but now it's back in `2022.10.4`
